### PR TITLE
Create racket-langserver.scrbl

### PR DESCRIPTION
**Pull request by spdegabrielle**
based on readme

### CODE REVIEW

File: scribblings/racket-langserver.scrbl [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+Next, install the package via @tt{raco}:
+
+@commandline{raco pkg install racket-langserver}
+
+Once it is installed, you can configure your editor to use a custom LSP client for Racket files (@tt{.rkt}), 
+and set the command for the custom client to
+
+@commandline{racket -l racket-langserver}
+
+You may need to restart your LSP runtime or your editor for `racket-langserver` to start.
```
Comment (sorawee) [inline scribblings/racket-langserver.scrbl:35]: The Markdown syntax should be changed to Scribble.

File: scribblings/racket-langserver.scrbl [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+@section{Installation and usage}
+
+A Racket runtime is a prerequisite, so before using @tt{racket-langserver}, ensure that a Racket runtime 
+is installed. You can install an from the @hyperlink["https://download.racket-lang.org"]{official download page}
+or install one from your package manager.
+
+@section{VSCode}
+
+Use the @hyperlink["https://marketplace.visualstudio.com/items?itemName=evzen-wybitul.magic-racket"]{Magic Racket} extension.
```
Comment (sorawee) [inline scribblings/racket-langserver.scrbl:20]: IIRC, VSCode requires users to run`raco pkg install racket-langserver` too, right? It feels weird to have the instructions in "Other editors and IDEs" but not here.

File: scribblings/racket-langserver.scrbl (hunk 1 above)
Comment (spdegabrielle) [reply scribblings/racket-langserver.scrbl]: done

**Review by dannypsnl — APPROVED** [review]

**Comment by jeapostrophe** [comment]
Thank you!

**Pull request merged**